filebeat/docs/filebeat-log-rotation.asciidoc

```diff
@@ -0,0 +1,91 @@
+[[input-file-log-rotation]]
+== Configure logrotation for the input files of {beatname_uc}
+
+Incorrectly configured logrotation can result in either lost log messages or duplicated events when using {beatname_uc} to
+forward messages. When you see missing or duplicate events when running the Beat, be sure to check how input files are rotated.
+
+[float]
+[[log-rotation-linux]]
+=== Logrotation on Linux
```

Comment:
Is this empty section correct?

I think the cases described here are also true for Windows. It's just more common on windows to have the logging application do the rotation.

Reply:
I removed the empty section and renamed the last section about Windows, so it shows that all of the above applies to Windows, too.